### PR TITLE
Addressing comments lab2 FSx lustre

### PR DESCRIPTION
**Pull request by jyothibv**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags = ["tutorial", "lustre", "FSx"]
 In this section, you will create a new FSx for lustre filesystem in your test account
 
 
-1. Click [here](https://console.aws.amazon.com/fsx/home) to navigate to your FSx console. 
+1. Open the FSx console [here] (https://console.aws.amazon.com/fsx/home) to navigate to your FSx console.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:11]:
```suggestion
1. Open the FSx console [here] (https://console.aws.amazon.com/fsx/home). 
```

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ tags = ["tutorial", "lustre", "FSx"]
 In this section, you will create a new FSx for lustre filesystem in your test account
 
 
-1. Click [here](https://console.aws.amazon.com/fsx/home) to navigate to your FSx console. 
+1. Open the FSx console [here] (https://console.aws.amazon.com/fsx/home) to navigate to your FSx console. 
 
 2. On the FSx console click on **Create file system** button as shown below
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:13]:
```suggestion
2. Choose **Create file system*, as shown below
```

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ tags = ["tutorial", "lustre", "FSx"]
...
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:14]: 3. are redundant. I cannot comment below.

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 4]
```diff
@@ -41,27 +39,27 @@ Please leave the cloud9 terminal open in another tab through this  lab.
...
 
 ![filesystemnetwork](/images/fsx-for-lustre-hsm/filesystemnetwork.png)
 
-7. Scroll down to the **Backup and Maintanence** section and disable automatic backups as shown below. This step is important because you will be creating a data repository association in section c and to do this we need to make sure automatic backups are disabled.
+6. Scroll down to the **Backup and Maintanence** section and disable automatic backups as shown below. This step is important because you will be creating a data repository association in section c and to do this we need to make sure automatic backups are disabled.
 
 ![disablebackup](/images/fsx-for-lustre-hsm/disablebackup.png) 
 
-8. Retain the rest of the details in the form at default settings and hit the **Next** button.
+7. Retain the rest of the details in the form at default settings and hit the **Next** button.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:50]:
```suggestion
7. Choose **Next**.
```

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 5]
```diff
@@ -41,27 +39,27 @@ Please leave the cloud9 terminal open in another tab through this  lab.
...
 
-9. On the next screen review the settings and at the end of the page as shown below hit the **Create file system** button.
+8. On the next screen review the settings and at the end of the page as shown below hit the **Create file system** button.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:52]:
```suggestion
8. Review the **Summary** and choose **Create file system**.
```

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 6]
```diff
@@ -41,27 +39,27 @@ Please leave the cloud9 terminal open in another tab through this  lab.
...
 
 ![createfsl](/images/fsx-for-lustre-hsm/createfsl.png)
 
-10. Creating this file system is expected to take around 10 mins. In the meantime let us move to next steps. 
+9. Creating this file system is expected to take around 10 mins. In the meantime let us move to next steps.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:56]:
```suggestion
9. Amazon FSx for Lustre file system creation will take ~10 mins. In the meantime let us move to next steps. 
```

File: content/08-fsx-lustre/02-creates3-upload.md [hunk 7]
```diff
@@ -55,7 +55,7 @@ aws s3 ls s3://$BUCKET_NAME_DATA/
 rm SEG_C3NA_Velocity.sgy
 ```
 
-7. Navigate to the S3 console via AWS console by clicking [here](https://console.aws.amazon.com/s3/home).
+7. If you would like a view of S3 through the UI open the Amazon S3 console [here](https://console.aws.amazon.com/s3/home).
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/02-creates3-upload.md:58]:
```suggestion
7. If you would like a view of S3 through the UI open the Amazon S3 console [here](https://console.aws.amazon.com/s3/home).
```
```suggestion
7. If you would like a view of S3 through the user interface, open the [Amazon S3 console](https://console.aws.amazon.com/s3/home).
```

File: content/08-fsx-lustre/04-launch-ec2.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ tags = ["tutorial", "lustre", "FSx", "S3"]
 
 In this section you will launch an Amazon EC2 instance which will be used in the next section to mount the filesystem created in section a.
 
-1. Click [here](https://console.aws.amazon.com/ec2/home) to navigate to your EC2 dashboard on AWS console.. 
+1. Navigate to the EC2 Dashboard [here](https://console.aws.amazon.com/ec2/home) to navigate to your EC2 dashboard on AWS console..
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/04-launch-ec2.md:10]:
```suggestion
1. Open [Amazon EC2 Console](https://us-east-1.console.aws.amazon.com/ec2/home?#Instances). 
```

File: content/08-fsx-lustre/04-launch-ec2.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ tags = ["tutorial", "lustre", "FSx", "S3"]
 
 In this section you will launch an Amazon EC2 instance which will be used in the next section to mount the filesystem created in section a.
 
-1. Click [here](https://console.aws.amazon.com/ec2/home) to navigate to your EC2 dashboard on AWS console.. 
+1. Navigate to the EC2 Dashboard [here](https://console.aws.amazon.com/ec2/home) to navigate to your EC2 dashboard on AWS console.. 
 
 2. On the EC2 dashboard click on **Launch instance** button
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/04-launch-ec2.md:12]:
```suggestion
2. Choose **Launch instances*.
```

File: content/08-fsx-lustre/_index.md [hunk 10]
```diff
@@ -14,3 +14,23 @@ In this lab, you are introduced to [Amazon FSx for Lustre](https://aws.amazon.co
...
+
+As a fully managed service, Amazon FSx makes it easier for you to use Lustre for workloads where storage speed matters. FSx for Lustre eliminates the traditional complexity of setting up and managing Lustre file systems, enabling you to spin up and run a battle-tested high-performance file system in minutes. It also provides multiple deployment options so you can optimize cost for your needs.
+
+FSx for Lustre is POSIX-compliant, so you can use your current Linux-based applications without having to make any changes. FSx for Lustre provides a native file system interface and works as any file system does with your Linux operating system. It also provides read-after-write consistency and supports file locking.
+
+### HSM and Data Repository with S3
+
+[HSM](https://en.wikipedia.org/wiki/Hierarchical_storage_management) also known as tiered storage,is a data storage and Data management technique that automatically moves data between high-cost and low-cost storage media.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/_index.md:29]:
```suggestion
Hierarchical Storage Management, [HSM](https://en.wikipedia.org/wiki/Hierarchical_storage_management) also known as tiered storage,is a data storage and Data management technique that automatically moves data between high-cost and low-cost storage media.
```

File: content/08-fsx-lustre/_index.md [hunk 11]
```diff
@@ -14,3 +14,23 @@ In this lab, you are introduced to [Amazon FSx for Lustre](https://aws.amazon.co
...
+
+When you use Amazon FSx with multiple durable storage repositories, you can ingest and process large volumes of file data in a high-performance file system by using automatic import and import data repository tasks. At the same time, you can write results to your data repositories by using automatic export or export data repository tasks. With these features, you can restart your workload at any time using the latest data stored in your data repository.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/_index.md:31]:
```suggestion
When you use Amazon FSx for Lustre with multiple durable storage repositories, you can ingest and process large volumes of file data in a high-performance file system by using automatic import and import data repository tasks. At the same time, you can write results to your data repositories by using automatic export or export data repository tasks. With these features, you can restart your workload at any time using the latest data stored in your data repository.
```

File: content/08-fsx-lustre/_index.md [hunk 12]
```diff
@@ -14,3 +14,23 @@ In this lab, you are introduced to [Amazon FSx for Lustre](https://aws.amazon.co
...
+
+Amazon FSx is deeply integrated with [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). This integration means that you can seamlessly access the objects stored in your Amazon S3 buckets from applications that mount your Amazon FSx file system. You can also run your compute-intensive workloads on Amazon EC2 instances in the AWS Cloud and export the results to your data repository after your workload is complete.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/_index.md:33]:
```suggestion
Amazon FSx is integrated with [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). This integration means that you can seamlessly access the objects stored in your Amazon S3 buckets from applications that mount your Amazon FSx file system. You can also run your compute-intensive workloads on Amazon EC2 instances in the AWS Cloud and export the results to your data repository after your workload is complete. 
```

**Review by mhuguesaws — COMMENTED** [review]
Very nice improvements over the first version. Left some comments and suggestion.

File: content/08-fsx-lustre/_index.md (hunk 12 above)
Comment (jyothibv) [reply content/08-fsx-lustre/_index.md:33]: Thank you for the feedback I have addressed all these suggestions in the updated pull request.

File: content/08-fsx-lustre/01-create-fsxl.md (hunk 3 above)
Comment (mhuguesaws) [reply content/08-fsx-lustre/01-create-fsxl.md:14]: @jyothibv could you address the comment above ?

**Review by mhuguesaws — APPROVED** [review]
I approve the PR but there is a numbering issue to fix.

File: content/08-fsx-lustre/07-cleanup-summary.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
++++
+title = "b. Cleanup and Summary"
+date = 2019-09-18T10:46:30-04:00
+weight = 80
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/07-cleanup-summary.md:4]: There is weight problem here

File: content/08-fsx-lustre/07-cleanup-summary.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
++++
+title = "b. Cleanup and Summary"
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/07-cleanup-summary.md:2]: Index letter is incorrect, should be g

**Pull request merged**